.jscsrc

```diff
@@ -1,48 +1,4 @@
 {
-  "disallowKeywords": ["with"],
-  "disallowKeywordsOnNewLine": ["else"],
-  "disallowMixedSpacesAndTabs": true,
-  "disallowMultipleVarDecl": "exceptUndefined",
-  "disallowNewlineBeforeBlockStatements": true,
-  "disallowQuotedKeysInObjects": true,
-  "disallowSpaceAfterObjectKeys": true,
-  "disallowSpaceAfterPrefixUnaryOperators": true,
-  "disallowSpacesInFunction": {
-      "beforeOpeningRoundBrace": true
-  },
-  "disallowSpacesInsideParentheses": true,
-  "disallowTrailingWhitespace": true,
-  "maximumLineLength": null,
-  "requireCamelCaseOrUpperCaseIdentifiers": true,
-  "requireCapitalizedComments": true,
-  "requireCapitalizedConstructors": true,
-  "requireCurlyBraces": true,
-  "requireSpaceAfterKeywords": [
-      "if",
-      "else",
-      "for",
-      "while",
-      "do",
-      "switch",
-      "case",
-      "return",
-      "try",
-      "catch",
-      "typeof"
-  ],
-  "requireSpaceAfterLineComment": true,
-  "requireSpaceAfterBinaryOperators": true,
-  "requireSpaceBeforeBinaryOperators": true,
-  "requireSpaceBeforeBlockStatements": true,
-  "requireSpaceBeforeObjectValues": true,
-  "requireSpacesInFunction": {
-      "beforeOpeningCurlyBrace": true
-  },
-  "requireTrailingComma": {
-      "ignoreSingleLine": true
-  },
-  "requireBlocksOnNewline": true,
-  "validateIndentation": 2,
-  "validateLineBreaks": "LF",
-  "validateQuoteMarks": "'"
+  "preset": "node-style-guide",
+  "maximumLineLength": null
```

Comment:
Actually, I forgot that **polyjuice** has a flag to report which rules have been discarded (the appropriately named `--discarded` or `-d`):

So here's what polyjuice can't convert from .jscsrc to .eslintrc rules:
```
$ polyjuice --jscs .jscsrc -d

disallowNewlineBeforeBlockStatements
requireCapitalizedComments
requireSpaceBeforeBinaryOperators
requireSpacesInFunction
requireBlocksOnNewline
```

So I say if those 5 rules _are_ important to you, we keep JSCS in the workflow (and publish a custom **jscs-preset-gulp** preset with those 5 rules; so we can reuse them between repos).

If those 5 rules _**aren't**_ important, let's remove the .jscsrc files and dependencies from the repos.